### PR TITLE
[VEN-2888]: TWT market to Core pool on BNB

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -143,6 +148,12 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
           symbol: "FDUSD",
           decimals: 18,
         },
+        {
+          isMock: true,
```
Comment (chechu) [inline helpers/deploymentConfig.ts:152]: I think we can reuse the MockTWT token deployed in the Isolated pool repo: `0xb99c6b26fdf3678c6e2aff8466e3625a0e7182f8`. wdyt?

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -162,6 +173,22 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
...
+          asset: "TWT",
+          symbol: "vTWT",
+          rateModel: InterestRateModels.JumpRate.toString(),
+          baseRatePerYear: "0",
+          multiplierPerYear: convertToUnit("0.2", 18),
+          jumpMultiplierPerYear: convertToUnit("3", 18),
+          kink_: convertToUnit("0.5", 18),
+          collateralFactor: convertToUnit("0.5", 18),
+          reserveFactor: convertToUnit("0.25", 18),
+          initialSupply: convertToUnit(100, 18), // TBD
```
Comment (chechu) [inline helpers/deploymentConfig.ts:187]: Let's set 5,000 TWT tokens for testnet

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -162,6 +173,22 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
           borrowCap: convertToUnit(4_400_000, 18),
           vTokenReceiver: vTreasuryAddress,
         },
+        {
+          name: "Venus TWT",
+          asset: "TWT",
+          symbol: "vTWT",
+          rateModel: InterestRateModels.JumpRate.toString(),
+          baseRatePerYear: "0",
```
Comment (chechu) [inline helpers/deploymentConfig.ts:181]:
```suggestion
          baseRatePerYear: convertToUnit("0.02", 18),
```

File: deployments/bsctestnet_addresses.json [hunk 4]
```diff
@@ -52,7 +53,7 @@
     "VBNBAdmin": "0x04109575c1dbB4ac2e59e60c783800ea10441BBe",
     "VBNBAdmin_Implementation": "0x90c891bb2b1821ADE159ECa6FbA0418b2bD1b86D",
     "VBNBAdmin_Proxy": "0x04109575c1dbB4ac2e59e60c783800ea10441BBe",
-    "VBep20Delegate": "0xad6aa8Bb4829560412A94AA930745f407BF8000B",
+    "VBep20Delegate": "0x228Ea224d62D14a2e2cb9B43083aE43954C39B67",
```
Comment (chechu) [inline deployments/bsctestnet_addresses.json:56]: Why do we have to redeploy a new `VBep20Delegate`?

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

File: helpers/deploymentConfig.ts [hunk 5]
```diff
@@ -162,6 +174,22 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
...
+          asset: "TWT",
+          symbol: "vTWT",
+          rateModel: InterestRateModels.JumpRate.toString(),
+          baseRatePerYear: convertToUnit("0.02", 18),
+          multiplierPerYear: convertToUnit("0.2", 18),
+          jumpMultiplierPerYear: convertToUnit("3", 18),
+          kink_: convertToUnit("0.5", 18),
+          collateralFactor: convertToUnit("0.5", 18),
+          reserveFactor: convertToUnit("0.25", 18),
+          initialSupply: convertToUnit(5_000, 18),
```
Comment (Debugger022) [inline helpers/deploymentConfig.ts:188]: Is initialSupply different from mainnet? [reference](https://github.com/VenusProtocol/venus-protocol/pull/527/files#diff-18185c86bd8baed36466d7a325efd11e7729ffdd43200a4808db76a6340185ffR237)

**Pull request merged**